src/backend/asset-status/index.ts

```diff
@@ -28,8 +28,8 @@ export function updateAssetStatusSS({
     const logger = Logger({ name: 'AssetStatusSSLib', logSetting: LOG_SETTING });
     const messaging = ClearBlade.Messaging();
 
-    function failureCb(reason: unknown): void {
-        logger.publishLog(LogLevels.ERROR, 'Failed ', reason);
+    function failureCb(error: Error): void {
+        logger.publishLog(LogLevels.ERROR, 'Failed ', error.message);
```

Comment:
I don't think you can guarantee that message is a property. For instance, MergeAsset rejects with a string literal.
I think a util function that checks for the message property would be good enough. Something like - 
```javascript
function getErrorMessage(error: unknown): string {
    if (typeof error === 'object' && error.message) {
        return error.message;
    } else {
        return JSON.stringify(error);
    }
}
```